src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -1998,4 +1998,40 @@
   <data name="UseConcreteTypeForParameterMessage" xml:space="preserve">
     <value>Change type of parameter '{0}' from '{1}' to '{2}' for improved performance</value>
   </data>
+  <data name="PreferLengthOverAnyCodeFixTitle" xml:space="preserve">
+    <value>Prefer 'Length' over 'Any()'</value>
+  </data>
+  <data name="PreferLengthOverAnyTitle" xml:space="preserve">
+    <value>Prefer 'Length' over 'Any()'</value>
+  </data>
+  <data name="PreferLengthOverAnyMessage" xml:space="preserve">
+    <value>Prefer a 'Length' check over 'Any()' to avoid the need for devirtualization</value>
+  </data>
+  <data name="PreferLengthOverAnyDescription" xml:space="preserve">
+    <value>Prefer a 'Length' check over an 'Any()' call. The 'Any()' call will convert the collection into an IEnumerable which creates the need for devirtualization at runtime.</value>
+  </data>
+  <data name="PreferCountOverAnyCodeFixTitle" xml:space="preserve">
+    <value>Prefer 'Count' over 'Any()'</value>
+  </data>
+  <data name="PreferCountOverAnyTitle" xml:space="preserve">
+    <value>Prefer 'Count' over 'Any()'</value>
```

Comment:
Probably add property to make it less confusing with[ `DoNotUseCountWhenAnyCanBeUsed`](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1827)
```suggestion
    <value>Prefer 'Count' property check over 'Any()'</value>
```